### PR TITLE
feat: adding style for toggle button

### LINKED ISSUE #16: Style - Move and Style Light/Dark Theme Toggle

### LINKED ISSUE #18: Functionality/Style - Allow for Toggle between Light/Dark Themes

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR adds style to the toggle button. 

## Issue
Closes #18 

## Information for Reviewers 

> [!NOTE]
> This is a work-in-progress

### CODE REVIEW

**Comment by CBID2** [comment]
/review
--pr_reviewer.num_code_suggestions="8"
--pr_reviewer.inline_code_comments=true

**Review by lexoskeleton — COMMENTED** [review]
I am having issues seeing the changes between light and dark modes on my machine - @audreyclarkdev @Iroh-Omolola @knopgm @NuraSad is anyone available to pull down issue-18 on and test?

File: start-here/js/script-main.js [hunk 1]
```diff
@@ -78,3 +78,8 @@ optionsList.forEach((option) => {
   }
   option.addEventListener("click", handler);
 });
+// Dark Mode Toggle
+function myFunction() {
+  var element = document.body;
+  element.classList.toggle("dark-mode");
+ }
```
Comment (NuraSad) [inline start-here/js/script-main.js]: I think you need to add an event listener for the checkbox and connect it to this function.

**Review by lexoskeleton — CHANGES_REQUESTED** [review]
see @NuraSad 's comment for more details & let us know if you need help!

File: start-here/js/script-main.js (hunk 1 above)
Comment (Iroh-Omolola) [reply start-here/js/script-main.js]: @CBID2  need help with it?

Comment (CBID2) [reply start-here/js/script-main.js]: > @CBID2  need help with it?

Hey @Iroh-Omolola, I haven't been feeling well for the past week, so I can definitely use your help.

File: start-here/styles/style.css [hunk 2]
```diff
@@ -157,3 +189,30 @@ label[for="serif"] {
...
+/* light mode theme starts here */
+@media (prefers-color-scheme: light){
+  :root {
+--bg-color: #fff;
+--text-color: #000;
+}
+}
+
+/* light mode theme ends here */
+}
```
Comment (NuraSad) [inline start-here/styles/style.css]: Extra curly brace

**Review by NuraSad — CHANGES_REQUESTED** [review]
Overall style change works on my machine. Good work! Please address requested change and let me know.

**Pull request merged**